staff/carme-casserras/spotify-template/index.sass

```diff
@@ -0,0 +1,105 @@
+
+$color: white
+$width: 100%
+%head-footer-share
+    display: flex
+    flex-direction: column
+    justify-content: center
+    align-items: center
+    
+body
+    background: black
+    width: 100%
+    height: 100%
+    margin: 0
+    padding: 0
+    font-family: Arial, Helvetica, sans-serif    
+
+.head
+    @extend %head-footer-share
+    // height: 100px
+    padding-top: 1em  
+    
+#spotify-logo
+    width: 40%
+    height: 40%   
+    margin-bottom: 1.5em
+      /* For tablets: */
+    @media screen and (min-width: 700px) 
+        width: 20%
+        height: 20%
+
+.button
+    border-radius: 1.2em
+    background: rgb(43, 199, 90)  
+    margin-top: 25px
+    padding: .5em 2em
+    color: $color
+    border: 0
+     /* For tablets: */
+    @media screen and (min-width: 700px) 
+        width: 25%
+        height: 25%
+
+%disk-share
+    display: flex
+    flex-direction: column
+    align-items: center
+    overflow-x: hidden
+
+%disk__title-share
+    display: grid
+    grid-template-columns: repeat(2, 1fr)
+    //position: relative
+    padding: .5em
+
+.disk
+    @extend %disk-share
+    
+    &__sentence   
+        width: 11em
+        font-size: 1.5em
+        font-weight: bold
+        color: $color
+        text-align: center
+
+    &__title
+        @extend %disk__title-share
+        /* For tablets: */
+        @media screen and (min-width: 700px) 
+            grid-template-columns: repeat(4, 1fr)
+
+    &__title > img                
```

Comment:
@carmecass with Sass, you just need to type &__title once, its children would be:
`& > img`, `& > div`,...